beacon-chain/state/state-native/state_trie.go

```diff
@@ -93,17 +93,33 @@ var denebFields = append(
 	types.HistoricalSummaries,
 )
 
+var electraFields = append(
+	denebFields,
+	types.LatestExecutionPayloadHeaderElectra,
+	types.DepositReceiptsStartIndex,
+	types.DepositBalanceToConsume,
+	types.ExitBalanceToConsume,
+	types.EarliestExitEpoch,
+	types.ConsolidationBalanceToConsume,
+	types.EarliestConsolidationEpoch,
+	types.PendingBalanceDeposits,
+	types.PendingPartialWithdrawals,
+	types.PendingConsolidations,
+)
+
 const (
 	phase0SharedFieldRefCount                     = 10
 	altairSharedFieldRefCount                     = 11
 	bellatrixSharedFieldRefCount                  = 12
 	capellaSharedFieldRefCount                    = 14
 	denebSharedFieldRefCount                      = 14
+	electraSharedFieldRefCount                    = 17
 	experimentalStatePhase0SharedFieldRefCount    = 5
 	experimentalStateAltairSharedFieldRefCount    = 5
 	experimentalStateBellatrixSharedFieldRefCount = 6
 	experimentalStateCapellaSharedFieldRefCount   = 8
 	experimentalStateDenebSharedFieldRefCount     = 8
+	experimentalStateElectraSharedFieldRefCount   = 8
```

Comment:
This is wrong, the value should be `11`. The name of the const is confusing, but it means "what is the number of `SharedFieldRefCount` when you enable the experimental state? The answer is the number without the experimental state minus multi-slice fields.

Reply:
can that be added in some comment in the code? i didn't understand the number changes either.